packages/langium-cli/src/generator/grammar-serializer.ts

```diff
@@ -41,9 +41,13 @@ export function serializeGrammar(services: LangiumCoreServices, grammars: Gramma
                 });
                 // The json serializer returns strings with \n line delimiter by default
                 // We need to translate these line endings to the OS specific line ending
-                const json = normalizeEOL(serializedGrammar
+                let json = normalizeEOL(serializedGrammar
                     .replace(/\\/g, '\\\\')
                     .replace(new RegExp(delimiter, 'g'), '\\' + delimiter));
+                if (!production) {
```

Comment:
WTF? Why not in production?

Reply:
@Lotes The issue isn't present in production mode

Reply:
Because the minifier uses `'` instead of template strings to embed the grammar in code. The `${` characters only result in issues in template strings.